src/types.ts

```diff
@@ -69,3 +66,38 @@ interface AragonEnvironment {
   registry: string
   wsRPC: string
 }
+
+/**
+ * App Installer types
+ */
+
+export type NetworkType =
+  | 'homestead'
```

Comment:
Is this mainnet? We probably just want `'mainnet'` or `'main'` (I believe this is what web3.js returns for mainnet) instead.

Reply:
Yes, this is mainnet. But we are using a ethers.js providers that use 'homested' for mainnet. We are using mainnet as default so I don't think is going to be a pain point for users.

Reply:
Gotcha, makes sense!